handbook/marketing/livestream.md

```diff
@@ -0,0 +1,117 @@
+# Livestream events (Webinars)
+## Overview
+The following information describes the planning and execution of a Sourcegraph livestream event. 
+
+## Livestream information
+
+All livestream events will require the following details for planning:
+
+- Title
+- Date/Time
+- Short Description
+- Speaker Name
+- Speaker Bio
+- Speaker Headshot
+- Livestram Link from Zoom or GotoWebinar
+- Audience
+
+Sourcegraph uses these web applications for running a livestream event:
+
+- GotoWebinar or Zoom:
+  * Manage registration and attendance
+  * Invitation email and reminders with livestram link and details
+  * Run and record the livestream
+- Hubspot
+  * Set up landing page for registration
+  * Manage invitation lists and store registration/attendance activities
+
+## Livestream planning tasks
+
+ time to livestream | activity  
+ --------------- | ---------------    
+ 1 - 2 months out | Identify livestream information 
+ 1 month out | Send out invitations 
+ 2 weeks out | Send out invitation #2 
+ 1 day - 1 week out | Send out email reminders 
+ 1-2 weeks out | Perform a rehearsal 
+ 1 day out | Test livestream link 
+ 30 minutes | Final soundcheck 
+ Post livestream | Send out thank you emails with link to recorded livestream
+
+### Video conference setup
+
+Create the livestream, configuring the following:
+
+- Set the schedule, host and details
+- Registration landing page form
+- Thanks for registering email which includes the livestream link
+
+Zoom required meeting configuration **IMPORTANT**
+
+- Require meeting password, use US formated date i.e. 032620
+- Enable Video for Host, disable on Participant
+- Allow audio on both telephone and computer
+- Disable "Enable join before host"
+- Enable "Mute participants upon entry"
+- Add Alternative Hosts
+
+Zoom advanced configuration during broadcast, under 'Advanced Sharing Options'
+- Enable "One participant can share at a time"
+- Enable "Only Host" under "Who can share?"
+
+Zoom account management > account settings
+- Disable "File Transfer"
+- Disable "Allow Removed Participants to Rejoin"
+
+### Announcement on customer's Sourcegraph instance
```

Comment:
Does the announcement message need to include the password?

Reply:
no - it we will be included with the livestream link, emailed to the registrant.